tests/integration_tests/functional/test_concurrency.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests  multiple microVMs work correctly when spawned simultaneously."""
+
+import time
+
+from framework import decorators
+
+import host_tools.network as net_tools
+
+NO_OF_MICROVMS = 20
+
+
+@decorators.test_context('ssh', NO_OF_MICROVMS)
+def test_run_concurrency(test_multiple_microvms, network_config):
+    """Check we can spawn multiple microvms."""
+    microvms = test_multiple_microvms
+    assert microvms
```

Comment:
I'm not so sure, but is this assert needed/useful?

Reply:
No, just a leftover that tried to check the fixture was constructed successfully.